src/OpenSage.Game/Gui/Apt/ActionScript/Value.cs

```diff
@@ -11,6 +11,7 @@ public enum ValueType
         Constant,
         Boolean,
         Integer,
+        UInteger, 
```

Comment:
Are you sure we require that additional type? It seems like we might get away with treating uints as itnegers in most places?

Reply:
There is an EA_PushLong in the opcodes, but it reads an unit with 32 bits. I'm not sure what will happen(if there are really any numbers larger than 2147483648), maybe this type will be removed later.